ui/page/send/layout.go

```diff
@@ -223,7 +224,9 @@ func (pg *Page) pageSections(gtx layout.Context, title string, showAccountSwitch
 }
 
 func (pg *Page) toSection(gtx layout.Context) layout.Dimensions {
-
+	usdExchangeSet := components.USDExchangeValue ==
+		pg.WL.Wallet.
+			ReadStringConfigValueForKey(dcrlibwallet.CurrencyConversionConfigKey)
```

Comment:
There is some form of delay in updating this setting when the page is opened. 
I believe moving this to the `handle()` should improve the performance.